superset/explore/form_data/commands/parameters.py

```diff
@@ -23,7 +23,8 @@
 @dataclass
 class CommandParameters:
     actor: User
-    dataset_id: int = 0
+    datasource_type: str = ""
+    datasource_id: int = 0
```

Comment:
I'm not sure why we have datasource_id as required with a default to 0, so I kept with the same pattern and set datasource_type to an empty string. It almost seems like we should have two different classes here when a key exists and when it doesn't.

Reply:
I think it's a good idea to break it into multiple classes 👍🏼

Reply:
I agree, but if it's ok, I'll put this into a different PR since it's currently one class, and outside this scope.